### PR TITLE
Update index page title and description

### PR DESCRIPTION
**Pull request by joaofnfernandes**
Updates /index.html so that it presents a better description when you bookmark the page, and in search engines.

/ping @docs.maintainers

### CODE REVIEW

**Comment by sanscontext** [comment]
Not a fan of this wording - it sort of implies that we, the docs team, don't know what Docker is. I'd prefer the "Docker Documentation" phrase you'd suggested in chat.

**Comment by joaofnfernandes** [comment]
Our index page is kind of strange, in the sense that it is an index page and also shows in the navigation menu.

The current proposal looks like this

screen shot 2016-10-31 at 12 02 53
(notice the browser tab)

My idea for "what is Docker" came from the website.
screen shot 2016-10-31 at 12 01 33

File: _data/toc.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 toc:
-- title: Welcome to the Docs
+- title: What is Docker?
```
Comment (johndmulhausen) [inline _data/toc.yaml]: Some possibilities: 

Docker Documentation
Documentation Home
(No entry at all) 

I think these are all better than a left-nav entry that says "What is Docker?" 

cc @sanscontext @londoncalling @mstanleyjones 

WDYT?

Comment (johndmulhausen) [inline _data/toc.yaml]: No entry is viable in the case of the home page; either way, the left-nav tree will be completely collapsed. The only difference is there wouldn't be a blue-colored nav entry on the left side.

Comment (mstanleyjones) [inline _data/toc.yaml]: I prefer "Docker Documentation" or "Docker Docs". From a quick google I found "AWS Documentation", "Oracle Help Center", "Microsoft Technical Documentation", "Guides and Sample Code - Apple Developer",

**Pull request closed without merging**